client-side/test/components/Report/htmlPDF.test.js

```diff
@@ -0,0 +1,68 @@
+// DownloadPage.test.js
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import html2canvas from 'html2canvas';
+import jsPDF from 'jspdf';
+import DownloadPage from '../../../src/components/Report/htmlPDF'; 
+import { LABEL_OF_PDF_DOWNLOAD_HTML } from '../../../src/components/Report/report.constant';
+
+// דמוי של הפונקציות החיצוניות
+jest.mock('html2canvas');
+jest.mock('jspdf');
+
+describe('DownloadPage', () => {
+  let addImageMock, saveMock;
+
+  beforeEach(() => {
+    addImageMock = jest.fn();
+    saveMock = jest.fn();
+
+    // דמוי של html2canvas שמחזיר Promise עם Canvas
```

Comment:
please remove all the Hebrew comments, or better - translate them to English - do it for all tests files